content/en/docs/Advanced/sys-extensions.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Extending the system with systemd extensions"
```

Comment:
I wonder if it would make sense here to say the UKI or USI instead of system. Not a strong opinion, just an observation

Reply:
we have moved to call it Trusted Boot so that works for me. Then ion the Trusted Boot page explains the UKI vs USI thingie